chef_master/source/chef_license_accept.rst

```diff
@@ -0,0 +1,277 @@
+
+#. Run the ``chef-<PRODUCT-NAME>-ctl reconfigure`` command using the ``--accept-license`` option. This automatically types ``yes`` and skips printing the Chef MLSA.
+
+#. Add a ``.license.accepted`` file to the ``/var/opt/<PRODUCT-NAME>/`` directory. The contents of this file do not matter. As long as this file exists in this directory, the Chef MLSA is accepted and the reconfigure process will not prompt for ``yes``.
+
+-----------------------------------------------------
+ Accept the Chef EULA
+-----------------------------------------------------
+
+Workstation Products
+=====================================================
+* Chef Workstation, which also contains
+    * Chef Infra
+    * Chef InSpec
+    * Push Jobs Client
+* Chef Infra
+* Chef InSpec
+* Habitat
+
+These products are typically installed on a user's workstation.
+Two methods are generally used to accept the license for these products:
+
+#. ``--chef-license <value>`` argument passed to the command line invocation.
+
+#. ``CHEF_LICENSE=<value>`` as an environment variable.
+
+``<value>`` can be specified as one of the following:
+
+#. ``accept`` - Accepts the license and attempts to persist a marker file locally. Persisting these marker files means future invocations do not require accepting the license again.
+
+#. ``accept-silent`` - Similar to ``accept`` except no messaging is sent to STDOUT
+
+#. ``accept-no-persist`` - Similar to ``accept-silent`` except no marker file is persisted. Future invocation will require accepting the license again.
+
+If no command line argument or environment variable is set, these products will attempt to request acceptance through an interactive prompt.
+If the prompt cannot be displayed, then the product will fail with an exit code 172.
+
+If the product attempts to persist the accepted license and fails, a message will be sent to STDOUT, but product invocation will continue successfully. 
+In a future invocation, however, the license would need to be accepted again.
+
+Products with specific features or differences from this general behavior are documented below.
+
+Chef Workstation
+-----------------------------------------------------
+Chef Workstation contains multiple chef products.
+When invoking the ``chef`` command line tool and accepting the license, users are required to accept the license for all the embedded products as well.
+The same license applies to all products, but each product must have its own license acceptance.
+``chef --chef-license accept`` will accept the license for Chef Workstation, Chef Infra, Chef InSpec, and Push Jobs Client.
+
+Chef Infra
+-----------------------------------------------------
+In addition to the above methods, users can specify ``chef_license = 'accept'`` in their Chef config.
+On a workstation, this can be specified in ``~/.chef/config.rb`` or ``~/.chef/knife.rb``, and on a node, it can be specified in ``/etc/chef/client.rb``.
```

Comment:
Can the license be accepted in the credentials file? We want to point people at credentials first and config.rb next. Knife.rb is mostly removed from the docs at this point.

Reply:
@tas50 Good point. Will address this point in next week's PR for this doc.